.github/actions/argus-builder/build-prep/action.yml

```
@@ -110,6 +118,17 @@ runs:
             core.setOutput('base', `${{ inputs.path_filters_base }}`);
           }
 
+    - name: Require manifest trigger label in pull requests
+      id: require_stack_label
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const labels = `${{ inputs.pr_labels }}`;
+          const triggerLabel = `${{ inputs.manifest_trigger_label }}`;
+          let hasTriggerLabel = labels.split(',').includes(triggerLabel)
```

Some apps need to call this directly instead of through the reusable workflow so let's add a `trim` to sanitize their input
```suggestion
          const hasTriggerLabel = labels.split(',').map(l => l.trim()).includes(triggerLabel)
```

---

done